tests/cms/aod/test_integrated_query.py

```diff
@@ -87,3 +87,14 @@ def test_isnonull_call():
         .Select(lambda muons: muons.Count())
     )
     assert training_df.iloc[0]['col1'] == 3
+
+
+def test_sumChargedHadronPt():
```

Comment:
This integrated test is great - but we also need a test or two that runs outside the integrated test pathway. Something that generates the C++ code, and then scans the actual code to see if the C++ member reference is put in correctly.

See the tests in `test_xaod_executor.py` for example.